src/Kiota.Builder/Writers/Java/CodeMethodWriter.cs

```diff
@@ -379,7 +379,7 @@ private static void WriteSetterBody(CodeMethod codeElement, LanguageWriter write
             writer.WriteLine($"this.{codeElement.AccessedProperty?.NamePrefix}{codeElement.AccessedProperty?.Name} = {value}");
         }
         else
-            writer.WriteLine($"this.{backingStore.Name}.set(\"{codeElement.AccessedProperty?.Name}\", value);");
+            writer.WriteLine($"this.{backingStore.Name.ToFirstCharacterLowerCase()}.set(\"{codeElement.AccessedProperty?.Name}\", value);");
```

Comment:
The name normalization should happen in the refiner for Java, @andreaTP invested some time to do so, let's not regress.

Reply:
Thanks 🙏